Libraryfile

```diff
@@ -1 +1 @@
-MIDDLE_VERSION=v0.44.0-nightly.20251016.1
```

Comment:
we will not merge this pr to develop until there will be proper middleware veriosn